packages/aws-cdk/test/api/deploy-stack.test.ts

```diff
@@ -210,7 +210,7 @@ test('deploy is skipped if template and tags did not change', async () => {
   expect(createChangeSetCalled).toBeFalsy();
   expect(executeChangeSetCalled).toBeFalsy();
   expect(describeStacksInput!).toStrictEqual({ StackName: 'withouterrors' });
-  expect(getTemplateInput!).toStrictEqual({ StackName: 'withouterrors', TemplateStage: 'Original' });
+  expect(getTemplateInput!).toStrictEqual({ StackName: 'withouterrors', TemplateStage: 'Processed' });
```

Comment:
ideally we add a fresh test for the new execution path. but here we're changing the core logic.

we don't recommend modifying original tests. do any of these break (are they using macros?)